requirements-full.txt

```diff
@@ -16,19 +16,19 @@ Products.MailHost==3.0
 Products.ZCTextIndex==4.0.1
 Products.ZCatalog==4.0.0
 Record==3.4
-RestrictedPython==4.0a1
+RestrictedPython==4.0a2
 WSGIProxy2==0.4.3
 WebOb==1.7.2
 WebTest==2.0.27
 ZConfig==3.1.0
 ZEO==5.1
-ZODB==5.2.3
+ZODB==5.2.4
 ZServer==4.0a2
 five.globalrequest==1.0
 five.localsitemanager==3.0.1
 funcsigs==1.0.2
 mock==2.0.0
-pbr==3.0.0
+pbr==3.0.1
```

Comment:
I tried removing it, but it's also used by `mock`, which in turn is a test dependency of `transaction`. Not sure why that wasn't an issue in the ZTK buildout